packages/api/src/cors.js

```diff
@@ -57,6 +57,9 @@ export function withCorsHeaders (handler) {
  * @returns {Response}
  */
 export function addCorsHeaders (request, response) {
+  // Clone the response so that it's no longer immutable
+  response = new Response(response.body)
```

Comment:
This throws away any headers set on the response already, including the status code.

Reply:
yes, I am still testing this